src/cow-react/modules/swap/containers/ImportTokenModal/index.tsx

```diff
@@ -46,10 +46,16 @@ export function ImportTokenModal(props: ImportTokenModalProps) {
           // Any token addresses that are loaded from the shorthands map do not need to show the import URL
           const supported = supportedChainId(chainId)
           if (!supported) return true
```

Comment:
Not part of this PR, I know, but this check doesn't need to be done inside the loop as it won't change with the token

Reply:
I agree, it looks weird, but the check is located inside of `filter` and the function returns an array of tokens.
The file should be refactored, I will add TODO